### PR TITLE
[Feature][Connector-v2] Support schema evolution for Oracle connector

### LINKED ISSUE #7204: [Feature][CDC] Support DDL parse for oracle-cdc
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Currently, we don't support read oracle ddl, does anyone want to add support for oracle?

reference
https://github.com/apache/seatunnel/pull/6929

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by dailai**
### Purpose of this pull request

Support schema evolution for oracle.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hailin0

File: docs/zh/concept/schema-evolution.md [hunk 1]
```diff
@@ -57,3 +61,90 @@ sink {
...
+    username = "dbzuser"
+    password = "dbz"
+    database-names = ["ORCLCDB"]
+    schema-names = ["DEBEZIUM"]
+    table-names = ["ORCLCDB.DEBEZIUM.FULL_TYPES"]
+    base-url = "jdbc:oracle:thin:@oracle-host:1521/ORCLCDB"
+    source.reader.close.timeout = 120000
+    connection.pool.size = 1
+    debezium {
+        include.schema.changes = true
```
Comment (hailin0) [inline docs/zh/concept/schema-evolution.md:87]: add `redo_log_catalog`

File: docs/zh/concept/schema-evolution.md [hunk 2]
```diff
@@ -57,3 +61,90 @@ sink {
...
```
Comment (hailin0) [inline docs/zh/concept/schema-evolution.md:132]: add `redo_log_catalog`

File: docs/zh/concept/schema-evolution.md (hunk 1 above)
Comment (dailai) [reply docs/zh/concept/schema-evolution.md:87]: done

**Review by hailin0 — APPROVED** [review]
LGTM

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**